README.md

```diff
@@ -53,6 +53,7 @@ The site specific parameters that this theme recognizes are:
 - `copyright` string: Custom copyright notice.
 - `math` boolean: Site wide kill switch for Latex support
 - `katex` boolean: if "katex" is set to true katex will be used to render LaTex, if not MathJax will be used instead
+- `codeBlocksDark` boolean: if true, code blocks will use a dark theme.
```

Comment:
I'm not sure that I will keep it for the first release, dark themed code block are not really in the "Tufte" spirit. But in the meantime it will stay.

Reply:
I agree. But the default was a dark theme and I didn't want to remove the functionality. By default (either explicitly set to false or not set at all) the light theme is used.